backend/src/modules/doubles-tournament/core/DoublesTournamentModuleCore.ts

```diff
@@ -12,20 +12,28 @@ import { FindAllDoublesTournaments } from './application/query/FindAllDoublesTou
 import { FindAllDoublesTournamentsQueryHandler } from './application/query/FindAllDoublesTournamentsQueryHandler';
 import { FindDoublesTournamentById } from './application/query/FindDoublesTournamentById';
 import { FindDoublesTournamentByIdQueryHandler } from './application/query/FindDoublesTournamentByIdQueryHandler';
+import { QueueMatch } from './application/command/QueueMatch';
+import { QueueMatchCommandHandler } from './application/command/QueueMatchCommandHandler';
+import { MatchesQueueRepository } from './application/MatchesQueueRepository';
 
 export function DoublesTournamentModuleCore(
   eventPublisher: DomainEventPublisher,
   commandPublisher: CommandPublisher,
   currentTimeProvider: CurrentTimeProvider,
   entityIdGenerator: EntityIdGenerator,
   repository: DoublesTournamentRepository,
+  matchesQueue: MatchesQueueRepository,
 ): ModuleCore {
   return {
     commandHandlers: [
       {
         commandType: CreateTournamentWithTeams,
         handler: new CreateTournamentWithTeamsCommandHandler(eventPublisher, currentTimeProvider, entityIdGenerator, repository),
       },
+      {
+        commandType: QueueMatch,
```

Comment:
Queue to kolejka. Enqueue to jest "zakolejkować" - jeśli dobrze pamiętam :)

Reply:
queue to też czasownik, ale racja jest coś takiego jak enqueue, które tutaj bardziej pasuje :-)